Ollivanders/src/net/pottercraft/Ollivanders2/House/O2HouseType.java

```diff
@@ -0,0 +1,158 @@
+package net.pottercraft.Ollivanders2.House;
+
+import org.bukkit.ChatColor;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * House types
+ *
+ * @author Azami7
+ * @since 2.2.8
+ */
+public enum O2HouseType
+{
+   GRYFFINDOR ("Gryffindor", ChatColor.DARK_RED),
+   HUFFLEPUFF ("Hufflepuff", ChatColor.GOLD),
+   RAVENCLAW ("Ravenclaw", ChatColor.DARK_BLUE),
+   SLYTHERIN ("Slytherin", ChatColor.DARK_GREEN);
+
+   private String name;
+   private String prefix;
+   private ChatColor color;
+   private Integer score;
+
+   private Map<ChatColor, String> colorPrefixes = new HashMap<ChatColor, String>() {{
+      put(ChatColor.DARK_BLUE, "§1");
```

Comment:
Is this missing black on purpose?

Reply:
Yes, black is too hard to see/read on the screen.